esp/src/src/nls/es/hpcc.ts

```diff
@@ -621,6 +639,7 @@ export = {
     PleaseSelectAUserOrGroup: "Por favor escoja un usario o grupo y nombre de archivo",
     PleaseSelectAUserToAdd: "Por favor escoja el usario para agregar",
     Plugins: "Complementos",
+    Pods: "Vainas",
```

Comment:
I think we're better off leaving this as pods, vaina is a negative word we should avoid. Also, docker seems to keep the term pod in their ES doc
https://kubernetes.io/es/docs/concepts/workloads/pods/